injector/injector.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 const (
-	emptyAdmResponse = `<VAST version="3.0"><Ad><Wrapper><AdSystem>prebid.org wrapper</AdSystem><VASTAdTagURI><![CDATA[%s]]></VASTAdTagURI><Creatives></Creatives></Wrapper></Ad></VAST>`
+	emptyAdmResponse = `<VAST version="3.0"><Ad><Wrapper><AdSystem>prebid.org wrapper</AdSystem><VASTAdTagURI><![CDATA[%s]]></VASTAdTagURI><Impression></Impression><Creatives></Creatives></Wrapper></Ad></VAST>`
```

Comment:
Why is this empty tag added? What happens if this is omitted?

Reply:
This is as per PRD to return empty Vast in case bid.Adm  is empty  and NURL is present.
[Events PRD](https://docs.google.com/document/d/1a-U1DytiCTtacM27qH3OZwhPa7bDwQzfpVRHDhyikWw/edit?tab=t.0#heading=h.tayl9cd5mdo2)